azure-pipelines.yml

```diff
@@ -11,36 +11,28 @@ variables:
 pool: 'Self Host Ubuntu'
 
 jobs:
-  - job: Build
+  - job: BuildTest
+    workspace:
+      clean: all
     steps:
       - template: azure-pipelines.tools.yml
 
-      # - template: azure-pipelines.cache.yml
-
       - script: |
           yarn
         displayName: yarn
 
+      - template: .devops/templates/pr-target-branch.yml
+
       - script: |
           yarn checkchange
         displayName: check change
 
       - script: |
-          yarn build
-        displayName: build
-
-      - task: ArchiveFiles@2
-        inputs:
-          rootFolderOrFile: $(Build.Repository.LocalPath)
-          # Include root folder so archive can just be extracted into $(Pipeline.Workspace)
-          includeRootFolder: true
-          archiveType: 'tar'
-          tarCompression: 'gz'
-          archiveFile: $(Build.ArtifactStagingDirectory)/build.tar.gz
-          replaceExistingArchive: true
-
-      - publish: $(Build.ArtifactStagingDirectory)
-        artifact: Build-PR-$(Build.BuildNumber)
+          yarn lage build test lint --verbose --no-cache
```

Comment:
The use of lage has the same benefit or better compared to doing these in separate jobs, right?